msdk-io-mgf/src/test/java/io/github/msdk/io/mgf/MgfFileImportMethodTest.java

```diff
@@ -0,0 +1,35 @@
+package io.github.msdk.io.mgf;
+
+import io.github.msdk.MSDKException;
+import java.io.File;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by evger on 17-May-18.
+ */
+public class MgfFileImportMethodTest {
+  @Test
+  public void testMgfImport() throws IOException, MSDKException {
+    final int expectedSize = 1;
+    final String expectedTitle = "example.9.9.2";
+    final long expectedNumberDatapoints = 18;
+    final int expectedCharge = 2;
+
+    File file = new File("target/test-classes/test_query.mgf");
+    MgfFileImportMethod importMethod = new MgfFileImportMethod(file);
+    Collection<MgfMsSpectrum> spectrums = importMethod.execute();
+    int size = spectrums.size();
+    MgfMsSpectrum spectrum = (MgfMsSpectrum)spectrums.toArray()[0];
```

Comment:
It is better to return List instead of Collection. Then you can call ````spectrums.get(0)```` instead of using this weird expression ````(MgfMsSpectrum)spectrums.toArray()[0]````